backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BitcodeCompiler.kt

```diff
@@ -146,7 +117,11 @@ internal class BitcodeCompiler(val context: Context) {
             }
             addNonEmpty(profilingFlags)
         }
-        targetTool("clang++", *flags.toTypedArray(), file, "-o", objectFile)
+        if (configurables is AppleConfigurables) {
```

Comment:
What's that?

Reply:
We use clang from Xcode distribution to avoid verification problems in the App Store and to use features that are not merged to the upstream LLVM yet.